simple_certmanager/admin.py

```diff
@@ -92,6 +94,51 @@ class SigningRequestAdmin(admin.ModelAdmin):
         ),
     )
 
+    def get_urls(self):
+        urls = super().get_urls()
+        my_urls = [
+            path(
+                "download_csr/<int:pk>",
```

Comment:
```suggestion
                "download_csr/<int:pk>/",
```
purely cosmetic :) but we like our trailing slashes :D